agenda-webapps/src/main/webapp/vue-app/agenda/components/event/form/AgendaEventFormDates.vue

```diff
@@ -46,11 +46,13 @@
           <div class="d-flex flex-nowrap v-event-draggable">
             <strong>{{ event.summary }}</strong>
             <date-format
```

Comment:
You can add all those DOM in one single condition :
<template v-if="!allDay">
  <date-format ...
 <strong...
  <date-format ...
</template>